x11-wm/enlightenment/enlightenment-0.22.3.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson xdg-utils
+
+DESCRIPTION="Enlightenment DR17 window manager"
+HOMEPAGE="https://www.enlightenment.org/"
+SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0.17/${PV%%_*}"
+KEYWORDS="~amd64 ~x86"
+
+e_conf_mods=(
+	applications bindings dialogs display
+	interaction intl menus paths
+	performance randr shelves theme
+	window-manipulation window-remembers
+)
+
+e_norm_mods=(
+	appmenu backlight battery bluez4
+	clock conf connman cpufreq
+	everything fileman fileman-opinfo gadman
+	geolocation ibar ibox lokker
+	luncher mixer msgbus music-control
+	notification packagekit pager pager-plain
+	quickaccess shot start syscon
+	sysinfo systray tasks teamwork
+	temperature tiling time vkbd
+	winlist wireless wizard wl-buffer
+	wl-desktop-shell wl-drm wl-text-input wl-weekeyboard
+	wl-wl wl-x11 xkbswitch xwayland
+)
+
+IUSE_E_MODULES=(
+	${e_conf_mods[@]/#/enlightenment_modules_conf-}
+	${e_norm_mods[@]/#/enlightenment_modules_}
+)
+
+IUSE="doc nls pam systemd udisks wayland ${IUSE_E_MODULES[@]/#/+}"
+
+DEPEND="
+	>=dev-libs/efl-1.20.5[X]
+	virtual/pkgconfig
+	virtual/udev
+	x11-libs/libXext
+	x11-libs/libxcb
+	x11-libs/xcb-util-keysyms
+	x11-misc/xkeyboard-config
+	pam? ( sys-libs/pam )
+	systemd? ( sys-apps/systemd )
+	udisks? ( sys-fs/udisks:2 )
+	wayland? (
+		dev-libs/efl[wayland]
+		>=dev-libs/wayland-1.11.0
+		>=x11-libs/libxkbcommon-0.3.1
+		>=x11-libs/pixman-0.31.1
+	)
+"
+
+src_prepare() {
+	default
+
+	# fix QA issues with .desktop files
+	find data/desktop/ -type f -exec sed -i 's|OnlyShowIn=Enlightenment|OnlyShowIn=X-Enlightenment|g' {} \; || die
```

Comment:
Does that actually work after changing it?

Reply:
But does enlightement actually show the item? I'm afraid changing the value would result in hiding it.

Reply:
I tested it with and without the find-sed line, and filemanager.desktop is displayed in both cases. [It has a specified category](https://git.enlightenment.org/core/enlightenment.git/tree/data/desktop/enlightenment_filemanager.desktop#n47) and is found under it, while [askpass has NoDisplay=true](https://git.enlightenment.org/core/enlightenment.git/tree/data/desktop/enlightenment_askpass.desktop#n13). However I did find that if you disable 'enlightenment_modules_fileman' it still installs this .desktop file, and it does nothing. Should I do some if-use case here to modify the file if fileman USE is enabled, and rm it if it isnt?

Reply:
File a bug upstream. I don't think it's a major enough for you to fix it in Gentoo immediately.

Reply:
In case someone still reads this, emixer.desktop was fixed:
https://git.enlightenment.org/core/enlightenment.git/commit/?id=ab1360c224a9991a04957edf4a647a328e2ca475

and the X-Enlightenment errors were reported already to Enlightenment devs, but it requires action from xdg-tools developers. I filed a bug earlier today, 
https://bugs.freedesktop.org/show_bug.cgi?id=106563